arcade/scene.py

```diff
@@ -29,6 +29,24 @@ def __init__(self) -> None:
         self._sprite_lists: List[SpriteList] = []
         self._name_mapping: Dict[str, SpriteList] = {}
 
+    def __len__(self) -> int:
+        """
+        Get length of `_sprite_lists`.
+        """
+        return len(self._sprite_lists)
+
+    def __delitem__(self, sprite_list: Union[str, SpriteList]) -> None:
+        """
+        Remove the SpriteList from `_sprite_lists` and `_name_mapping`.
+
+        :param Union[str, SpriteList] sprite_list: which SpriteList to delete - can
+        be either the name of the SpriteList or the SpriteList object.
+        """
+        if isinstance(sprite_list, str):
+            self.remove_sprite_list_by_name(sprite_list)
```

Comment:
Maybe you could add an is instance check for an integer, because I'm sure some users will want that. Like: a animal simulator.

Reply:
Maybe you could make a self.remove_sprite_list_by_index.

Reply:
Too late I guess now.

Reply:
I would definitely be open to a follow up PR adding a case for integers to remove based on the list index.

Reply:
@tiffanyxiao do you want to do this? If you don't I'll do it.

Reply:
Sure, I'll try it!